static/app/styles/tokens.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: Tokens
+description: Tokens are the smallest unit of a Design System. They represent discrete design decisions in a platform-agnostic way.
+status: in-progress
+resources:
+  figma: https://www.figma.com/design/97XKAiOW0Rg8Rk1hb02R4E/%F0%9F%90%A6-Primitives?node-id=446-34701&p=f&t=secYuPMqzV9RoH8F-0
+  js: https://github.com/getsentry/design-engineering/tree/main/packages/scraps/src/tokens
+  reference:
+    DTCG Glossary: https://www.designtokens.org/glossary/
+    Design Tokens Format Module: https://www.designtokens.org/tr/third-editors-draft/format/
+    Design Tokens explained: https://atlassian.design/tokens/design-tokens
+---
+
+import {useTheme} from '@emotion/react';
+import color from 'color';
+
+import {Container, Flex} from 'sentry/components/core/layout';
+import {Text} from 'sentry/components/core/text';
+import * as Storybook from 'sentry/stories';
+
+# Color
+
+## Categories
+
+Colors have been organized into separate scales for specific use cases.
+
+| Category     | Description                                                                                            |
+| ------------ | ------------------------------------------------------------------------------------------------------ |
+| `background` | Painting an elements background.                                                                       |
+| `content`    | Painting text and icons which need to stand out the most.                                              |
+| `graphics`   | Painting icons and other graphical elements which don’t need to stand out as much as `content` tokens. |
+| `borders`    | Painting borders of containers or visual diagrams.                                                     |
+| `shadows`    | Painting shadows of containers.                                                                        |
+| `controls`   | Painting interactive elements and all their states.                                                    |
+| `syntax`     | Painting `<code/>` elements that depend on PrismJS.                                                    |
+| `focus`      | Painting `:focus` or validation states.                                                                |
```

Comment:
@natemoo-re @Jesse-Box, some feedback, but I find focus here to be a bit out of place. We already have controls, which is described by `Painting interactive elements and all their states`, so I assume focus would be covered there?

Reply:
@JonasBa fully agree. Can't remember if we discussed it on Notion, but I had the same though and am working on combining them! If not already covered, `focus` can be a new section in `controls`.